### PR TITLE
Implement a command that lists ongoing schedules

### PR DESCRIPTION
**Pull request by sarjinius**

### CODE REVIEW

File: src/test/java/scm/address/logic/parser/AddressBookParserTest.java [hunk 1]
```diff
@@ -258,6 +257,14 @@ public void parseCommand_listScheduleCommand() throws Exception {
         assertTrue(parser.parseCommand(ListScheduleCommand.COMMAND_WORD + " 5") instanceof ListScheduleCommand);
     }
 
+    @Test
+    public void parseCommand_listOngoingScheduleCommand() throws Exception {
```
Comment (tahsinhasem) [inline src/test/java/scm/address/logic/parser/AddressBookParserTest.java:261]: Consider changing function name to (command)_(input)_(expectation i.e:success/fail/exception) form? so it'd be parseCommand_listOngoingScheduleCommand_success() ?

File: src/main/java/scm/address/logic/commands/ListOngoingScheduleCommand.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ */
+public class ListOngoingScheduleCommand extends Command {
+    public static final String COMMAND_WORD = "list_ongoing_schedule";
+
+    public static final String MESSAGE_SUCCESS = "Listed all ongoing schedules!";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredScheduleList(new DuringDateTimePredicate(Optional.of(LocalDateTime.now())));
```
Comment (tahsinhasem) [inline src/main/java/scm/address/logic/commands/ListOngoingScheduleCommand.java:22]: Maybe consider adding these in new lines. As you are instantiaiting multiple classes and might be difficult to read. Perhaps Something like:
```
LocalDatimeTIme dt = ...
DuringDateTimePredicate predicate= ...
model.updateFilteredScheduleList(predicate)
```

would be more readable!

**Review by tahsinhasem — CHANGES_REQUESTED** [review]
Overall Looks good to me. But I've suggested some small changes!

File: src/test/java/scm/address/logic/parser/AddressBookParserTest.java (hunk 1 above)
Comment (sarjinius) [reply src/test/java/scm/address/logic/parser/AddressBookParserTest.java:261]: I think the current name is good enough, what do you think? It is consistent with the other function names in the class.

File: src/main/java/scm/address/logic/commands/ListOngoingScheduleCommand.java (hunk 2 above)
Comment (sarjinius) [reply src/main/java/scm/address/logic/commands/ListOngoingScheduleCommand.java:22]: Would do so, thanks!

Comment (tahsinhasem) [reply src/main/java/scm/address/logic/commands/ListOngoingScheduleCommand.java]: Awesome!

**Review by tahsinhasem — APPROVED** [review]
Looks Good!

**Pull request merged**